transform-chunk-sizes.ts

```diff
@@ -1,36 +1,39 @@
-import { Buffer } from "./deps.ts";
```

Comment:
Unfortunately something about this changed `TransformChunkSizes` is causing the integration tests to fail, which means there must be a bug in how it's processing the data.

> Then I discovered There is a better alternative to Buffer available in every runtime: https://sindresorhus.com/blog/goodbye-nodejs-buffer

That article is about Node's `Buffer`. This repo uses the [Deno `Buffer`](https://github.com/denoland/deno_std/blob/main/io/buffer.ts) which explicitly [says](https://github.com/denoland/deno_std/blob/283888de59f9bb6e795714b5a5229b061521be21/io/buffer.ts#L21-L23):

> * Buffer is NOT the same thing as Node's Buffer. Node's Buffer was created in
> * 2009 before JavaScript had the concept of ArrayBuffers. It's simply a
>  * non-standard ArrayBuffer.

What is wrong with using [Deno's `Buffer` implementation](https://github.com/denoland/deno_std/blob/main/io/buffer.ts)? It doesn't use any Deno-specific APIs as far as I know. If you just leave it in, what doesn't work?

BTW at one point Deno std decided to deprecate their `Buffer`, so I actually created a branch of this that works without it, by copying its code into this repo: https://github.com/bradenmacdonald/deno-s3-lite-client/compare/no-buffer . But they [changed their mind](https://github.com/denoland/deno_std/pull/4184), and so I'm still using Deno std Buffer.

Reply:
I wasn't sure dnt will make it work, I can try